Library/Homebrew/cask/audit.rb

```diff
@@ -510,12 +510,13 @@ def audit_signing
           when Artifact::Pkg
             system_command("spctl", args: ["--assess", "--type", "install", path], print_stderr: false)
           when Artifact::App
-            system_command("spctl", args: ["--assess", "--type", "execute", path], print_stderr: false)
+            system_command("syspolicy_check", args: ["distribution", path], print_stderr: false)
```

Comment:
Do we want to fall back on older macOS systems?

Reply:
Either gate to Sonoma or `which("syspolicy_check")`